optimize/util/optimize-commons/pom.xml

```diff
@@ -199,16 +211,20 @@
       <version>1.12.772</version>
       <scope>compile</scope>
     </dependency>
+    <!-- unused/declared
     <dependency>
       <groupId>software.amazon.awssdk</groupId>
       <artifactId>sdk-core</artifactId>
       <version>${awssdk.version}</version>
     </dependency>
+    -->
+    <!-- unused/declared
     <dependency>
       <groupId>software.amazon.awssdk</groupId>
       <artifactId>sts</artifactId>
       <version>${awssdk.version}</version>
     </dependency>
+    -->
```

Comment:
I think this was added with OpenSearch in mind recently. Would be good to check with Giuliano before we delete this

Reply:
Tagging @grlimacan

Reply:
This needs to stay unfortunately, this is the weird way that the AWS integration works, which is that if those packages are present it knows it needs to use it. I am not sure how exactly this is handled in the background by AWS

Reply:
Ok then I'll re-include it and I'll add it to the ignore set.
Thanks.